codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSSigningParams.kt

```diff
@@ -52,7 +52,7 @@ class SigV4Configurator(
         useDoubleURIEncode = params.service.sdkId.lowercase(Locale.US) != "s3",
         useURLPathNormalization = params.service.sdkId.lowercase(Locale.US) != "s3",
         useUnsignedPayload = params.operation.hasTrait<UnsignedPayloadTrait>() || params.forceUnsignedBody,
-        useSignedBodyHeader = listOf("s3", "glacier").contains(params.service.sdkId.lowercase(Locale.US)) &&
+        useSignedBodyHeader = listOf("s3", "glacier", "s3 control").contains(params.service.sdkId.lowercase(Locale.US)) &&
```

Comment:
I can't remember off-hand, is sdkId the service identifier that is guaranteed not to change?
(I thought that one didn't have spaces in it)

Reply:
Yep it is! And it allows spaces

https://smithy.io/2.0/aws/aws-core.html#sdkid
- "The value MUST NOT change once a service is publicly released. If the value does change, the service will be considered a brand new service in the AWS SDKs and Tools."
- "... the value can only contain alphanumeric characters and spaces. ... and when using spaces, each space must be between two alphanumeric characters."